ravenframework/ROMExternal.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
Hm, if there is a better name than ROMExternal (or ROMLoader), I would be happy to change it.

Reply:
I don't know of anything better offhand. Since it's being used externally we might consider RavenROM or something? Maybe we can make a poll for the RAVEN team.

Reply:
The other change I contemplated is putting it inside of ravenframework.CustomDrivers, since it sorta is a driver.

Reply:
hmmmm, yeah, I could see that. Maybe that's something that should be looked at in the future, as the existing drivers all use RAVEN workflows, whereas this tool wraps a single model.

Reply:
I don't see much movement on the name front, so I think we can merge this and change it later if we need to.